metaserver/src/main/scala/scala/meta/languageserver/CompilerConfig.scala

```diff
@@ -0,0 +1,51 @@
+package scala.meta.languageserver
+
+import java.io.File
+import java.nio.file.Files
```

Comment:
Why are we using IO and NIO?

Reply:
java.io is only for `File.pathSeparator`, otherwise we use nio for I/O work

Reply:
NIO Filesystem has a platform independent separator if you want to stick with the NIO api.

Reply:
I think I prefer java.io.File.pathSeparator over FileSystems.getDefault.getSeparator